src/components/Modals/NewCourse.vue

```diff
@@ -5,13 +5,13 @@
     <div class="autocomplete">
       <input class="newCourse-dropdown" :id="'dropdown-' + semesterID" :ref="'dropdown-' + semesterID" :placeholder="placeholder" @keyup.enter="addCourse" @keyup.esc="closeCourseModal" />
     </div>
-    <div v-if="!isOnboard && !selected"> <!-- if a course is not selected -->
+    <!-- <div v-if="!isOnboard && !selected"> 
```

Comment:
Can you leave a TODO here just explaining why this is commented out? I assume you'll prob be working on it later but it doesn't hurt